File: cobd/time.py
#Nathaniel Castro Time of Day 


time = int(input('What time is it for you?: '))



if time <= 11:
    print("Good morning")
    
if time >= 12 and time <= 18:
    print("Good afternoon")

if time >= 20:
    print("Good night")

#else:
#    print("Go to bed NOW!")

File: cobd/time.c
//Nathaniel Castro Time of Day C

#include <stdio.h>
int time;


int main(){
    printf("What time is it for you?: ");
    scanf("%d\n", &time);

        if (time <= 11){
            printf("Good morning\n");
        
        }else if (time >= 12, time <= 20){
            printf("Good afternoon\n");
        }else if (time >= 20){
            printf("Good night\n");
        }


    return 0;
}